maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -9,27 +9,46 @@
 import org.apache.maven.plugin.AbstractMojo;
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Component;
 import org.apache.maven.plugins.annotations.Mojo;
 import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.maven.plugins.annotations.ResolutionScope;
+import org.apache.maven.project.MavenProject;
+import org.apache.maven.project.ProjectDependenciesResolver;
+import org.eclipse.aether.RepositorySystem;
+import org.eclipse.aether.RepositorySystemSession;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import aQute.bnd.build.Workspace;
+import aQute.bnd.maven.lib.resolve.DependencyResolver;
 import aQute.bnd.osgi.Constants;
+import aQute.bnd.repository.fileset.FileSetRepository;
 import aQute.bnd.service.RepositoryPlugin;
 import biz.aQute.resolve.Bndrun;
 
 @Mojo(name = "export", defaultPhase = PACKAGE, requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
 public class ExportMojo extends AbstractMojo {
 	private static final Logger	logger	= LoggerFactory.getLogger(ExportMojo.class);
 
+	@Parameter(defaultValue = "${project}", readonly = true, required = true)
+	private MavenProject				project;
+
+	@Parameter(defaultValue = "${repositorySystemSession}", readonly = true, required = true)
+	private RepositorySystemSession		repositorySession;
+
 	@Parameter(readonly = true, required = true)
 	private List<File>	bndruns;
 
 	@Parameter(defaultValue = "${project.build.directory}", readonly = true)
 	private File		targetDir;
 
+	@Parameter(readonly = true, required = false)
+	private List<File>					bundles;
```

Comment:
Actually, I think we need to change how we configure these plugins. Instead of allowing multiple bndruns, we allow only a single bndrun and you use once execution per bndrun if you need more. Then you can specify different bundles for each bndrun.

Old way:
```
            <plugin>
                <groupId>biz.aQute.bnd</groupId>
                <artifactId>bnd-export-maven-plugin</artifactId>
                <version>${bnd.version}</version>
                <configuration>
                    <failOnChanges>false</failOnChanges>
                    <bndruns>
                        <bndrun>mylaunch.bndrun</bndrun>
                        <bndrun>otherlaunch.bndrun</bndrun>
                    </bndruns>
                    <bundles>
                       <bundle>xxxxx</bundle>
                       <bundle>yyyy</bundle>
                       <bundle>zzz</bundle>
                    </bundles>
                </configuration>
                <executions>
                    <execution>
                        <goals>
                            <goal>export</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
```

New way:

```
            <plugin>
                <groupId>biz.aQute.bnd</groupId>
                <artifactId>bnd-export-maven-plugin</artifactId>
                <version>${bnd.version}</version>
                <configuration>
                    <failOnChanges>false</failOnChanges>
                </configuration>
                <executions>
                    <execution>
                       <id>mylaunch</id>
                       <goals>
                            <goal>export</goal>
                       </goals>
                       <configuration>
                         <bndrun>mylaunch.bndrun</bndrun>
                         <bundles>
                            <bundle>xxxxx</bundle>
                            <bundle>yyyy</bundle>
                         </bundles>
                      </configuration>
                    </execution>
                    <execution>
                       <id>otherlaunch</id>
                       <goals>
                            <goal>export</goal>
                       </goals>
                       <configuration>
                         <bndrun>otherlaunch.bndrun</bndrun>
                         <bundles>
                            <bundle>zzz</bundle>
                            <bundle>yyyy</bundle>
                         </bundles>
                      </configuration>
                    </execution>
                </executions>
            </plugin>
```

I think this is cleaner for normal one bndrun configuration (Don't need <bndruns>) and allow better control for <bundles> for each bndrun when you have multiple bndruns.